cradle-core/src/main/java/com/exactpro/cradle/serialization/SerializationUtils.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2021-2022 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.cradle.serialization;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+
+import static com.exactpro.cradle.serialization.Serialization.INVALID_MAGIC_NUMBER_FORMAT;
+
+public class SerializationUtils {
+	
+	public static final int USHORT_MAX_VALUE = 65535;
+
+	static void printBody(byte[] body, ByteBuffer buffer) {
+		if (body == null) {
+			buffer.putInt(-1);
+		} else {
+			buffer.putInt(body.length);
+			buffer.put(body);	
```

Comment:
Can body be empty here? Won't this cause issue?

Reply:
Yes, it can.